joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -350,6 +350,12 @@ controller_interface::return_type JointTrajectoryController::update(
       }
     }
   }
+  else
```

Comment:
I would move this `else` before the `if` since it's just a few lines. Best to get it out of the way quickly for readability, rather than requiring a couple hundred lines of scrolling.